platform/navy/hooks/before_bro/before_bro.sh

```diff
@@ -34,7 +34,7 @@ fi
 
 if [[ -n "$VPN_IP" ]]; then
   sudo echo "" >>  /usr/local/bro/share/bro/site/local.bro
-#  sudo echo "redef restrict_filters += [[\"not-vpn\"] = \"not (port $VPN_PORT && host $VPN_IP && ip proto $VPN_PROTOCOL)\"];" >> /usr/local/bro/share/bro/site/local.bro
+  sudo echo "redef restrict_filters += [[\"not-vpn\"] = \"not ($VPN_PROTOCOL src port $VPN_PORT and src host $VPN_IP) and not ($VPN_PROTOCOL dst port $VPN_PORT and dst host $VPN_IP)\"];" >> /usr/local/bro/share/bro/site/local.bro
```

Comment:
This is a typical pitfall for `sudo` with IO redirection, where you can get around with `bash -c`.
And what's more,  HERE document might be used here to avoid escaping of double quotes
```
sudo bash -c 'cat >> /usr/local/bro/share/bro/site/local.bro' <<EOS
redef restrict_filters += [["not-vpn"] = "not ($VPN_PROTOCOL src port $VPN_PORT and src host $VPN_IP) and not ($VPN_PROTOCOL dst port $VPN_PORT and dst host $VPN_IP)"];
EOS
```